packages/ng/dialog/dialog-routing/dialog-routing.component.ts

```diff
@@ -87,15 +87,23 @@ export class DialogRoutingComponent<C> implements OnInit {
 
 	#ref?: LuDialogRef<C>;
 
-	async ngOnInit(): Promise<void> {
+	ngOnInit(): void {
```

Comment:
isn't this breaking?

Reply:
It could break some manual `ngOnInit().then` calls but it would be catched by TS compilation